module3/homework/passwordCheck/validation.cpp

```diff
@@ -1,2 +1,52 @@
 #include "validation.hpp"
-// TODO: Put implementations here
\ No newline at end of file
+#include <algorithm>
+
+bool doesPasswordsMatch(const std::string &password, const std::string &repeatedPassword) {
+    if (password.compare(repeatedPassword)) {
+        return false;
+    }
+    return true;
+}
```

Comment:
The simpliest way to solve this is:
```suggestion
   return password == repeatedPassword
}
```

Reply:
I agree, comparing two variables will  return true or false